gravity-forms/gw-multi-file-merge-tag.php

```diff
@@ -48,9 +48,10 @@ public function get_default_args() {
 			'form_id'        => false,
 			'field_ids'      => array(),
 			'exclude_forms'  => array(),
-			'default_markup' => '<li><a href="{url}">{filename}.{ext}</a></li>',
+			'default_markup' => '<a href="{url}">{filename}.{ext}</a>',
```

Comment:
Do we want to no longer provide this at all as a default?

Reply:
I removed the <li> wrapper from the default markup as a I realized we do not output the files in a <ul> by default and none of the file-type-specific default markup used <li>'s either.

There's an argument that we should default to <li>'s for everything since GF's default output for Multi File merge tags is wrapped in <ul> and <li>'s but given the lack of customer feedback about this during the lifetime of the snippet, I went with the least intrusive adjustment.

Thoughts?

Reply:
That works for me! We can always adjust.